mautic/mautic
# Fix getSecondaryIdentifier for Lead

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
We've had a issue in Lead entity with getSecondaryIdentifier for ages.
This PR fixed it and get back to column list the company name bellow the primary identifier.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create two contacts - first without firstname/lastname + company, second with company and firstname lastname
3. Check If  you see something like this after PR patch (the company is bellow the name with small font)
4. If the company is in the columns, then secondary identifier is not  visible (set in Configurations > Contacts settings)

![image](https://user-images.githubusercontent.com/462477/100340275-39dc1500-2fdb-11eb-8529-0e159b1de94f.png)
